<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFull</class>
 <widget class="QWidget" name="FFull">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список файлов ППСИ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/1/ftp.png</normaloff>:/res/1/ftp.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Каналы</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="cb1k">
             <property name="text">
              <string>1-й канал</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb2k">
             <property name="text">
              <string>2-й канал</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Состояние</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="cbObr">
             <property name="text">
              <string>Обработка завершена</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbNeobr">
             <property name="text">
              <string>Не обработанные</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbOch">
             <property name="text">
              <string>Ошибки</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Даты</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Начало отбора</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="de1">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Конец отбора</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="de2">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="cbDat">
             <property name="text">
              <string>Включить отбор по датам</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>№ НПН:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbNpn"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="cbOtbNpn">
            <property name="text">
             <string>Включить отбор по НПН</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>№ КА:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbNka">
            <item>
             <property name="text">
              <string>все</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>126</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>127</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Протоколы обработки</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="rbKvi">
             <property name="text">
              <string>ДДС КВИ</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbKvno">
             <property name="text">
              <string>ДДС КВНО</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbVrv">
             <property name="text">
              <string>ВРВ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnListProt">
             <property name="text">
              <string>Просмотр</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/res/1/data_grid.png</normaloff>:/res/1/data_grid.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btnRefresh">
          <property name="text">
           <string>Обновить</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/Sync.png</normaloff>:/res/Sync.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnVigr">
          <property name="text">
           <string>Выгрузить</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/1/folder_go.png</normaloff>:/res/1/folder_go.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStat">
          <property name="text">
           <string>Статистика</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/Stock graph.png</normaloff>:/res/Stock graph.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="text">
           <string>Редактировать</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/1/table_edit.png</normaloff>:/res/1/table_edit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDel">
          <property name="text">
           <string>Удалить</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/1/bin.png</normaloff>:/res/1/bin.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnKonv">
          <property name="text">
           <string>Конвертировать</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/1/data_sort.png</normaloff>:/res/1/data_sort.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="painGraf">
          <property name="text">
           <string>Графики</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/res/1/chart_curve.png</normaloff>:/res/1/chart_curve.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="pb">
          <property name="value">
           <number>24</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
